dali/base/daclient.cpp

```diff
@@ -102,6 +140,25 @@ bool initClientProcess(IGroup *servergrp, DaliClientRole role, unsigned mpport,
     initCoven(covengrp,NULL,clientVersion, minServerVersion);
     covengrp->Release();
     queryLogMsgManager()->setSession(myProcessSession());
+
+    // auto install environment monitor for server roles
+    // causes any config update hooks (installed by installConfigUpdateHook() to trigger on an env. change)
+    switch (role)
+    {
+        case DCR_ThorMaster:
+        case DCR_EclServer:
+        case DCR_EclAgent:
+        case DCR_SashaServer:
+        case DCR_DfuServer:
+        case DCR_EspServer:
+        case DCR_Roxie:
+        case DCR_AgentExec:
+        case DCR_EclScheduler:
+        case DCR_EclCCServer:
```

Comment:
DCR_DaliServer ?

Reply:
ok, so only clients

Reply:
yes, but some cli tools and legacy tools etc. are also excluded from this monitoring process